### ISSUE TITLE
[Playground]: H1 level heading is not defined in 'Playground' page.[Sev2][WCAG2.4.6][Auto]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#16]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.Actual:
3) Navigate to 'Playground' link and activate it.
4) Now, Turn on the 'Headings' toggle button and observe the issue.

**Actual:**
No H1 level heading is defined in 'Playground' page.
  
**Expected:**
H1 heading should be defined in the 'Playground' page.

**User Experience:**
Heading elements that are properly nested help users of assistive technologies understand the structure of the information on the web page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels
<img width="960" alt="H1 level heading is not defined in 'Playground' page" src="https://user-images.githubusercontent.com/85154417/121009056-54285400-c7b1-11eb-91c8-8f5c7c803097.png">

**Comment by apostolisms** [comment]
[This](https://github.com/microsoft/tsdoc/pull/303) PR fixes the issue.
